framework/src/auxkernels/ExtraElementIDAux.C

```diff
@@ -7,14 +7,14 @@
 //* Licensed under LGPL 2.1, please see LICENSE for details
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
-#include "ElemExtraIDAux.h"
+#include "ExtraElementIDAux.h"
 
-registerMooseObject("MooseApp", ElemExtraIDAux);
+registerMooseObject("MooseApp", ExtraElementIDAux);
```

Comment:
add the registerRenamed here

Reply:
Added. This actually makes Griffin pass for this PR although I still need to patch it.